Plugin/Magento/Framework/App/AreaList.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\UpwardConnector\Plugin\Magento\Framework\App;
+
+use Magento\Framework\App\Config\ScopeConfigInterface;
+use Magento\Store\Model\ScopeInterface;
+
+class AreaList
+{
+    /**
+     * @var ScopeConfigInterface
+     */
+    protected $scopeConfig;
+
+    /**
+     * Controller or frontname to load from default magento frontend
+     */
+    public const UPWARD_CONFIG_PATH_FRONT_NAMES_TO_SKIP = 'web/upward/front_names_to_skip';
+
+    /**
+     * @param ScopeConfigInterface $scopeConfig
+     */
+    public function __construct(ScopeConfigInterface $scopeConfig) {
+        $this->scopeConfig = $scopeConfig;
+    }
+
+    /**
+     * @param \Magento\Framework\App\AreaList $subject
+     * @param string|null $result
+     * @param string $frontName
+     * 
+     * @return string|null
+     */
+    public function afterGetCodeByFrontName(
+        \Magento\Framework\App\AreaList $subject,
+        $result,
+        $frontName
+    ) {
+
+        if ($result != 'frontend') {
+            return $result;
+        }
+
+        $frontNamesToSkip = explode(
+            "\r\n",
```

Comment:
Novice question: does Magento always store newlines in Windows format? Would `PHP_EOL` be a better option here?

Reply:
I made this change because it logically made sense, but need to double-check config values are saved in the same manner.

Reply:
Went ahead and rolled this back; even though my local `PHP_EOL` is `\n`, Magento was still saving these values as hard-coded Windows line breaks `/r/n`.